docker/jenkins.yml

```diff
@@ -7,11 +7,6 @@ credentials:
           id: "aws"
           scope: GLOBAL
           secretKey: "${/mgmt/secret_key}"
```

Comment:
Can this key also be removed, or are we still using keys rather than roles for some things?

Reply:
Don't know. Will need to check with Sam.

Reply:
We do need it. The ECS plugin we use needs an access key and secret to work and this is it. These should be the only AWS credentials we'll ever need though.